packages/wrangler/src/__tests__/middleware.test.ts

```diff
@@ -748,3 +748,68 @@ describe("unchanged functionality when wrapping with middleware", () => {
 		});
 	});
 });
+
+describe("multiple middleware", () => {
+	it("should respond correctly with D1 databases, scheduled testing, and formatted dev errors", async () => {
```

Comment:
For future reference, this describe block needs a

```ts
	runInTempDir();
```

To ensure that the `writeFileSync()` call doesn't pollute the source tree during tests.